### ISSUE TITLE
#-1953802417------------- All page content should be contained by landmarks

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#1853]
<strong>PageName:</strong> FeedDetailsPage  

<strong>Source:</strong>`<div id="attr-label-solar_lat" class="sc-dUWDJJ eBsayS" aria-describedby="calcite-tooltip-bb7a276d-64fc-8c9e-4319-54b346397953">solar_lat</div>`

<strong>Element Location: </strong>#attr-label-solar_lat

<strong>Impact: </strong>moderate

<strong>Failure Summary:</strong> Fix any of the following:
  Some page content is not contained by landmarks

<strong>Tags:</strong> cat.keyboard,best-practice

<strong>HelpUrl:<strong>https://dequeuniversity.com/rules/axe/4.6/region?application=axeAPI
<strong>Screenshot:<strong>`C:\Automation\Workspace\Velocity\Screenshot\FeedDetailsPage\-1953802417_All page content should be contained by landmarks.png`
